test/failure_handler/src/share/conf/common.properties

```diff
@@ -64,13 +64,13 @@ jstack.args=-e -l %p
 jstack.params.repeat=6
 
 jhsdb.app=jhsdb
-jhsdb.jstack.args=jstack --mixed --pid %p
-jhsdb.jstack.params.repeat=6
+jhsdb.jstack.live.args=jstack --mixed --pid %p
```

Comment:
What is the plan to address the fact that `--mixed` is not supported on macOS (and produces an error), so on macOS it would be best to leave `--mixed` off so we at least get java stack traces.